pkg/mcs/scheduling/server/apis/v1/api.go

```diff
@@ -126,6 +127,15 @@ func (s *Service) RegisterAdminRouter() {
 	router.DELETE("cache/regions/:id", deleteRegionCacheByID)
 }
 
+// RegisterConfigRouter registers the router of the config handler.
+func (s *Service) RegisterConfigRouter() {
+	router := s.root.Group("config")
+	router.GET("", getConfig)
+	router.GET("/schedule", getScheduleConfig)
+	router.GET("/replicate", getReplicationConfig)
+	router.GET("/store", getStoreConfig)
```

Comment:
I was wondering if we need these APIs, I think they are barely used.

Reply:
Because api server has the same interface. Maybe we also call "/config" in api server for these interfaces?

Reply:
I think it's better, WDYT?

Reply:
I think we can cancel "/config/schedule" and "/config/replicate", which are in "/config". But we should keep "config/store", otherwise we have no way to get store config, which is not in "/config".

Reply:
Or we can put store config into "/config" and add a new struct for it. How about it?
```
type ConfigPayload{
   Schedule
   Replication
   StoreConfig
   ...
}
```

Reply:
OK, make sense to me.

Reply:
Or only remove "/config/store", I found there is no "/config/store" in api server